### PR TITLE
Improved gyro initialisation

### PR DESCRIPTION
**Pull request by martinbudden**

### CODE REVIEW

**Comment by martinbudden** [comment]
@digitalentity , this is some further cleanup of the gyro initialisation which brings it into line with some changes I made in betaflight a while ago.

It doesn't directly address the memory problem, but I'd be interested to know if that problem still exists after these changes.

File: src/main/sensors/acceleration.c [hunk 1]
```diff
@@ -42,6 +42,7 @@
 acc_t acc;                       // acc access functions
 int32_t accADC[XYZ_AXIS_COUNT];
 sensor_align_e accAlign = 0;
+const uint32_t accTargetLooptime = 1000;
```
Comment (digitalentity) [inline src/main/sensors/acceleration.c]: we can't assume acc is sampled at 1khz rate. it's sampled at the same rate as gyro at the moment.

**Comment by stronnag** [comment]
I tried this on top of master. It was unflyable with massive vibrations on 'roll' --- elvis on steroids.

Log file:  http://seyrsnys.myzen.co.uk/inav-456/LOG00342.TXT
selection_484

**Comment by martinbudden** [comment]
@stronnag , can you try this again when you get a chance?

There was a silly mistake in the filter initialisation which I have now fixed.

**Comment by stronnag** [comment]
Tested. Works, no unexpected behaviours. Has no effect on #431.

**Comment by digitalentity** [comment]
@martinbudden can you address the merge conflict?

**Comment by martinbudden** [comment]
@digitalentity , this is rebased and ready to merge now

**Pull request merged**